frontend/src/views/CellCards/components/CellCard/index.tsx

```diff
@@ -36,100 +41,127 @@ export default function CellCard(): JSX.Element {
   const sectionRef2 = React.useRef(null);
   const sectionRef3 = React.useRef(null);
 
-  const [skinnyMode, setSkinnyMode] = useState<boolean>(true);
+  const [skinnyMode, setSkinnyMode] = useState<boolean>(false);
   // cell type id
   const { cellTypeId: cellTypeIdRaw } = router.query;
   const cellTypeId = (cellTypeIdRaw as string)?.replace("_", ":") ?? "";
   const cellTypesById = useCellTypesById() ?? {};
   const cellTypeName = cellTypesById[cellTypeId] ?? "";
 
+  const handleResize = useCallback(() => {
+    setSkinnyMode(
+      window.innerWidth < BREAKPOINT_WIDTH + 2 * LEFT_RIGHT_PADDING_PX
+    );
+  }, []);
+
+  const throttledHandleResize = useMemo(() => {
+    return throttle(handleResize, 100);
+  }, [handleResize]);
+
   useEffect(() => {
-    const handleResize = () =>
-      setSkinnyMode(
-        window.innerWidth < BREAKPOINT_WIDTH + 2 * LEFT_RIGHT_PADDING_PX
-      );
-    handleResize();
-    window.addEventListener("resize", handleResize);
+    throttledHandleResize();
+    window.addEventListener("resize", throttledHandleResize);
 
-    return () => window.removeEventListener("resize", handleResize);
-  }, []);
+    return () => window.removeEventListener("resize", throttledHandleResize);
+  }, [throttledHandleResize]);
+
+  const [geneInfoGene, setGeneInfoGene] = useState<Gene["name"] | null>(null);
```

Comment:
instead of using a reducer, a state here is enough to control whether to show the GeneInfoPanel or not